build.sbt

```diff
@@ -39,7 +41,6 @@ lazy val mimaSettings = mimaDefaultSettings ++ Seq(
 lazy val commonSettings = mimaSettings ++ Seq(
   organization := "com.typesafe.play",
   scalaVersion := scala212,
-  crossScalaVersions := Seq(scala212, scala211),
```

Comment:
Maybe that could be left there, but override crossScalaVersions for Java projects with `Seq(scala212)`?

Reply:
Hmm, that's what the setting was before this change, i.e. `Seq(scala212)`. I thought because of https://github.com/sbt/sbt-release/issues/219 that maybe I should try and avoid setting the value at all. Of course, it still has a value defined in `sbt.Defaults`. I'll admit that I haven't tested this because I haven't made a release.

Question: is there a way to dry-run a release?

Reply:
I don't know much about sbt-release, but I think you can substitute `+publishSigned` part with `+publishLocal` or something to test.